src/main/java/local/pbaranowski/zadaniejdk17/SimpleClassName.java

```diff
@@ -0,0 +1,7 @@
+package local.pbaranowski.zadaniejdk17;
+
+public interface SimpleClassName {
```

Comment:
Hmm.... tak raczej się w Javie nie robi :) 
Podejrzewam że chciałeś zaimplementować coś takiego jak mixin / trait w innych językach ale tu jest trochę inaczej - w tym celu powinna powstać klasa utilowa ze statycznymi metodami. :)

Reply:
Tak, chciałem zrobić coś w rodzaju minimalistycznego "wstrzykiwania metody", bardziej w zasadzie jako proof of concept ale ostatecznie zostawiłem, bo byłem ciekaw czy to akceptowalna praktyka :)